developers/weaviate/starter-guides/compression.mdx

```diff
@@ -0,0 +1,149 @@
+- [Binary Quantization (BQ)](/developers/weaviate/configuration/compression/bq-compression) BQ reduces the size of each vector dimension to a single bit. It works best for vectors with high dimensionality.
+
+## Compression considerations
+
+Before choosing a compression method, consider the [underlying index](#underlying-index). The index type determines which algorithms you can use. Some compression methods aren't available with some index types.
+
+The vectorizer that you use to create your object vectors may also limit your compression choices. For example, some embedding models are tuned specifically for BQ compression.
+
+Performance and cost are also important considerations. See [Cost, recall, and speed](#cost-recall-and-speed) for details.
+
+### Underlying index
+
+This table shows which compression algorithm is available for each index type.
+
+| Compression type | HNSW index | Flat index | Dynamic index |
+| :- | :- | :- | :- |
+| PQ | Yes | No | Yes |
+| SQ | Yes | No | Yes |
+| BQ | Yes | Yes | Yes |
+
+The [dynamic index](/developers/weaviate/config-refs/schema/vector-index#dynamic-indexes) is new in v1.25. This type of index is a [flat index](/developers/weaviate/config-refs/schema/vector-index#flat-indexes) until a collection reaches a threshold size. When the collection grows larger than the threshold size, the default is 10,000 objects, the collection is automatically reindexed and converted to an HNSW index.
+
+### Cost, recall and speed
+
+Performance includes speed and recall. In real world systems, these factors have to be balanced against cost. As you develop familiarity with your application, you can tune Weaviate to match your project and budget requirements.
+
+#### Cost
+
+These compression algorithms all help to control costs the same way. They reduce the size of the vector indexes. Smaller indexes need fewer resources so you spend less money.
+
+Compressed indexes use less RAM when they are loaded in memory, however they also use more disk space than uncompressed vectors. Weaviate stores the uncompressed vector and the compressed vector index. This means increased disk storage. However, since the cost of RAM is orders of magnitude higher than the cost of disk, the overall cost to use a compressed index is much lower than the cost of using an uncompressed index.
+
+The cost savings are obvious with in memory indexes such as HNSW. The greater the RAM reduction, the smaller the costs.
+
+- Uncompressed data size is roughly 7x greater than PQ compressed data.
+- Uncompressed data size is roughly 4x greater than SQ compressed data.
+- Uncompressed data size is roughly 32x greater than BQ compressed data.
+
+For additional details, refer to the [blog posts](#blog-posts).
```

Comment:
hmmm... I am not sure if this adds much at this point.
Maybe this sentence could be added at the start of the blog post segment 🤔